portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>sehhaj portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>get to know all about sehhaj</h1>
+      <img src="./images/avatar.png" style="width:50%;height:50%;">
+      <p>hi internet! nice to meet yall &#x1F44B;</p>
+      <p>i'm sehhaj! i just finished my third year at UNSW studying a bioinformatics engineering and commerce degree. 
+           slightly interesting facts about me, i have never cut my hair so it is really long. 
+           my middle name is dev which means god, but also I think maybe my parents were manifesting a potential future for me as a software dev? 
+           i am currently learning HTML as a STEP intern at Google. These are my favourite fruits and vegetables. &#x1F352;&#x1F351;&#x1F33D;&#x1F952;&#x1F347;</p>
```

Comment:
Again, you should be able to just include the characters rather than the &...; references.

Reply:
Removed references, includes just the characters now